docs/spire.md

```diff
@@ -0,0 +1,284 @@
+<!--
+---
+linkTitle: "SPIRE"
+weight: 1660
+---
+-->
+# SPIRE TaskRun result attestations
+
+The SPIRE TaskRun result attestations feature provides the non-falsifiable provenance to the build processes that run in the pipeline. They ensure that the results of the tekton pipeline executions originate from the build workloads themselves and that they have not been tampered. 
```

Comment:
nit: the first sentence makes it seem like result attestations alone deliver non-falsifiable provenance. can you clarify that this is one part of achieving non-falsifiable provenance, or that it guarantees non-falsifiable results specifically?

Reply:
Good catch. Changed.